src/commands/junit/upload.ts

```diff
@@ -264,3 +290,5 @@ UploadJUnitXMLCommand.addOption('basePaths', Command.Rest({required: 1}))
 UploadJUnitXMLCommand.addOption('maxConcurrency', Command.String('--max-concurrency'))
 UploadJUnitXMLCommand.addOption('logs', Command.Boolean('--logs'))
 UploadJUnitXMLCommand.addOption('rawXPathTags', Command.Array('--xpath-tag'))
+UploadJUnitXMLCommand.addOption('skipGitMetadataUpload', Command.Boolean('--skip-git-metadata-upload'))
+UploadJUnitXMLCommand.addOption('gitRepositoryURL', Command.String('--git-repository-url'))
```

Comment:
is this field now mandatory? Can't we read it from the git metadata extraction step?

Reply:
is that logic in this PR? As far as I can tell, `this.gitRepositoryURL` is always being used when calling `uploadToGitDB`. I might be missing something though

Reply:
oh got it, thanks